x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -22,6 +22,7 @@ export const allowedExperimentalValues = Object.freeze({
   riskyUsersEnabled: false,
   pendingActionResponsesWithAck: true,
   policyListEnabled: true,
+  policyResponseInFleetEnabled: false,
```

Comment:
Ahh. Ok. So the Fleet UI extension is not behind a flag - so anyone can use it. Only the Endpoint integration policy response use of it is behind a feature flag

Reply:
Yes, exactly, the extension point is there but we don't add the Policy Response component if the FF is not enabled. I did it in tis way because all the experimental features logic lives in security solution plugin but maybe there is another way to do it, thoughts?

Reply:
no no. this is good. I probably should have deleted my comment on here after understanding that only the endpoint use of the hook was behind a FF.

you're good 👍